### ISSUE TITLE
Update pip package dependencies/installation

### ISSUE THREAD

**Issue by boergens** [talmolab/sleap#1427]
# Bug description
Trying to run sleap-label brings the error message of the title and causes the program to close

Setup: Ubuntu 22.04 LTS
python3 -m venv env
source env/bin/activate
pip install sleap
sleap-label

gives

Saving config: /home/angie/.sleap/1.3.1/preferences.yaml
Traceback (most recent call last):
  File "/home/angie/sleap/env/bin/sleap-label", line 8, in <module>
    sys.exit(main())
  File "/home/angie/sleap/env/lib/python3.10/site-packages/sleap/gui/app.py", line 1631, in main
    window = MainWindow(
  File "/home/angie/sleap/env/lib/python3.10/site-packages/sleap/gui/app.py", line 174, in __init__
    self._initialize_gui()
  File "/home/angie/sleap/env/lib/python3.10/site-packages/sleap/gui/app.py", line 293, in _initialize_gui
    self._create_video_player()
  File "/home/angie/sleap/env/lib/python3.10/site-packages/sleap/gui/app.py", line 308, in _create_video_player
    self.player = QtVideoPlayer(
  File "/home/angie/sleap/env/lib/python3.10/site-packages/sleap/gui/widgets/video.py", line 231, in __init__
    self.seekbar = VideoSlider()
  File "/home/angie/sleap/env/lib/python3.10/site-packages/sleap/gui/widgets/slider.py", line 171, in __init__
    self.setAlignment(QtCore.Qt.AlignLeft | QtCore.Qt.AlignTop)
TypeError: 'PySide2.QtCore.Qt.AlignmentFlag' object cannot be interpreted as an integer

**Comment by roomrys** [comment]
Hi @boergens,

I see you tried the recommended method of installation (via [conda package](https://sleap.ai/installation.html#conda-package)) in #1428 without luck - I am working on building a new package with stricter requirements.

Does [conda install from source](https://sleap.ai/installation.html#conda-from-source) work fine for you?

Thanks,
Liezl

**Comment by boergens** [comment]
Hi, Thank you for the hint, conda install from source worked.
I used micromamba.
It went looking for '/some/prefix/envs/sleap/' so I had to create the folder `/some`
and micromamba threw this error message
`ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
rich 13.5.1 requires markdown-it-py>=2.2.0, but you have markdown-it-py 1.1.0 which is incompatible.
`
but now sleap-label is loading :-)
